docs/docs/k8s/ingress.md

```diff
@@ -162,12 +162,13 @@ The remaining annotations are specific to or behave differently than they do whe
 
 | Annotation                                            | Description                                                                                                                                                                                   |
 | ----------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `ingress.pomerium.io/tls_custom_ca_secret`            | Name of Kubernetes `tls` Secret containing a custom [CA certificate][`tls_custom_ca_secret`] for the upstream.                                                                                |
-| `ingress.pomerium.io/tls_client_secret`               | Name of Kubernetes `tls` Secret containing a [client certificate][tls_client_certificate] for connecting to the upstream.                                                                     |
-| `ingress.pomerium.io/tls_downstream_client_ca_secret` | Name of Kubernetes `tls` Secret containing a [Client CA][client-certificate-authority] for validating downstream clients.                                                                     |
-| `ingress.pomerium.io/secure_upstream`                 | When set to `"true"`, use `https` when connecting to the upstream endpoint.                                                                                                                   |
+| `ingress.pomerium.io/default-cert-secret`             | When a certificate is provided here in the format `namespace/secret_name`, it will be used as the default certificate for the Ingress.                                                                                                                                                                                              |
```

Comment:
this is not an `Ingress` resource annotation. This is an annotation for `IngressClass`, that allows you to skip providing `Ingress.spec.TLS.secret`

Reply:
@wasaga thanks for catching this. Where would a user apply it, and where should we document it?

Reply:
`IngressClass` is currently not documented as it is created by the Helm chart. 

The spec is rather simple: 
```yaml
apiVersion: networking.k8s.io/v1
kind: IngressClass
metadata:
  annotations:
    ingress.pomerium.io/default-cert-secret: pomerium/wildcard-localhost-pomerium-io
  name: pomerium
spec:
  controller: pomerium.io/ingress-controller
```

I suppose here we should document the existence of such annotation on the ingress class, and refer the reader to the Helm documentation reference for further detail.